packages/google-cloud-access-approval/tests/system/smoke_test.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import pytest
+
+from google.cloud import accessapproval_v1
+
+
+@pytest.fixture(scope="session")
+def project_id():
+    return os.environ["PROJECT_ID"]
+
+
+@pytest.mark.parametrize("transport", ["grpc", "rest"])
+def test_list_approval_requests(project_id: str, transport: str):
+    client = accessapproval_v1.AccessApprovalClient(transport=transport)
+
+    parent = client.common_project_path(project_id)
+    approval_requests = list(client.list_approval_requests(parent=parent))
+
+    assert len(approval_requests) >= 0
```

Comment:
Just to confirm: the idea is that if the smoke test fails, we won't reach this line (i.e. we won't silently get a zero-length list)?

Reply:
Clarified the purpose of the smoke test in [4ebbaf5](https://github.com/googleapis/google-cloud-python/pull/11735/commits/4ebbaf58da40fa7c58da3b3dde83ccf8ef5cb4dd)